common/kernel.c

```diff
@@ -37,6 +37,8 @@ extern int usermode_call_asm(user_func_t fn, void *fn_arg, unsigned long ret2ker
 
 void ret2kern_handler(void) {
     asm volatile("mov %%gs:(%0), %%" STR(_ASM_SP)::"r"(offsetof(percpu_t, ret2kern_sp)));
+
```

Comment:
Nit: I would merge the two inline asm calls into a single one

Reply:
Updated